frontend/src/containers/MetricsStep/Advance/Advance.tsx

```diff
@@ -77,25 +78,30 @@ export const Advance = () => {
               </IconButton>
             </StyledTooltip>
           </TooltipContainer>
-          <Link underline='none'>how to set up?</Link>
         </TitleAndTooltipContainer>
       </AdvancedContainer>
 
       {open && (
-        <AdvancedForm>
-          {fields.map(({ key, col, value }, index) => (
-            <TextField
-              variant='standard'
-              sx={{ gridColumn: `span ${col}` }}
-              key={index}
-              label={key}
-              value={value}
-              data-testid={key}
-              inputProps={{ 'aria-label': `input ${key}` }}
-              onChange={(e) => updateField(key, e.target.value)}
-            />
-          ))}
-        </AdvancedForm>
+        <>
+          <AdvancedForm>
+            {fields.map(({ key, col, value }, index) => (
+              <TextField
+                variant='standard'
+                sx={{ gridColumn: `span ${col}` }}
+                key={index}
+                label={key}
+                value={value}
+                data-testid={key}
+                inputProps={{ 'aria-label': `input ${key}` }}
+                onChange={(e) => updateField(key, e.target.value)}
+                placeholder={'Customized filed key'}
+              />
+            ))}
+          </AdvancedForm>
+          <Link underline='none' href={url} target='_blank' rel='noopener'>
+            how to setup
```

Comment:
`how to setup` -> `How to setup` 
![image](https://github.com/au-heartbeat/Heartbeat/assets/8743692/73036fe5-8741-4aa4-9ec8-8ddcc4257a44)